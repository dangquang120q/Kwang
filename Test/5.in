7
96
6439 2618 290 4965 3512 7917 5314 1268 6820 7964 9893 2707 3146 7429 6847 7472 339 4021 3445 3024 8464 2090 150 7182 7577 9358 7550 4398 4270 4987 7394 8530 6014 4487 3697 6539 9360 4121 1277 6822 1737 7514 8399 7402 4559 998 2634 6057 429 7703 7673 5750 314 8054 9572 4857 6521 5624 9160 6431 412 4495 8392 8031 9356 3846 4304 5915 4571 8423 4008 1200 893 6119 7585 5648 8275 4454 9076 2829 3851 1065 6427 909 4569 676 9112 2887 3822 5091 3804 998 7365 4425 7214 1293 
46
1979 3147 270 3685 2265 3886 7679 9470 6191 8541 5891 4099 7492 4368 7238 298 6996 8901 3389 9784 6176 2142 8841 4327 9608 7807 3561 8488 8795 432 9872 5950 2636 2642 5296 37 5489 1478 2285 3537 693 5482 6147 2428 3813 7941 
80
1125 2495 4394 5801 8178 6892 5295 4074 5136 5548 5410 8446 1228 4114 4286 1882 8861 4464 3189 2651 8036 1613 6823 1067 2954 5903 1031 1782 3153 1438 7775 780 2234 7457 1389 4214 9181 5259 6330 7842 2733 6849 1883 1128 9309 8601 9139 8788 5292 7996 1424 6154 5469 4217 498 453 7582 4530 473 7871 3297 653 6803 7400 5860 8390 4895 5405 1751 380 1761 1320 7956 5437 2287 2244 4839 4812 6658 1034 
26
9937 6306 8822 2453 5909 9350 3095 3442 5018 1965 577 4113 4397 8984 3544 7122 4476 9667 155 2854 5623 2984 2732 198 2115 6719 
97
4924 4181 2829 5954 2579 9898 7081 9811 4151 3965 9225 2521 3782 8482 910 1573 6386 8632 2463 6364 35 7355 4496 3319 7858 4654 5399 4180 5142 4326 7952 5952 3567 7897 6723 3628 5388 5019 7758 3944 5088 4086 1154 3025 4516 1728 1795 3762 7784 2015 3852 6854 2521 4102 4600 7695 81 9916 6331 1051 9961 8194 9123 3624 4330 8332 850 7897 4660 6844 8058 7921 7106 6851 8453 4809 5241 6128 482 4044 1179 7372 9361 1894 199 8483 7810 4002 3118 1530 9915 3581 2009 7282 3853 6401 1907 
13
5582 9730 3654 6962 7186 8514 6047 2295 9354 671 908 9328 2560 
45
1281 6791 4851 6590 8677 9498 7323 8427 7719 2813 4398 8119 3584 237 1984 7847 337 1102 2328 3953 4549 7991 1065 6595 6764 3591 8879 7070 8282 7405 5452 6296 6562 9301 4302 5994 6827 9440 1555 8177 2477 1987 9612 4786 1938 
