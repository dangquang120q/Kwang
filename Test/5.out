28825
29264
27629
28954
29792
28412
28550
